nsxt/resource_nsxt_policy_project.go

```diff
@@ -220,31 +209,28 @@ func resourceNsxtPolicyProjectPatch(connector client.Connector, d *schema.Resour
 		obj.ShortId = &shortID
 	}
 
-	vdsIntface, ok := d.GetOk("vpc_deployment_scope")
-	if util.NsxVersionHigherOrEqual("9.1.0") && ok && len(vdsIntface.([]interface{})) > 0 && vdsIntface.([]interface{})[0] != nil {
+	if util.NsxVersionHigherOrEqual("9.1.0") {
 		// There should be just one object here
-		vdScope := vdsIntface.([]interface{})[0].(map[string]interface{})
 		var spanReferences []model.SpanReference
-		if _, ok := vdScope["default_span_path"]; ok && vdScope["default_span_path"] != "" {
-			defaultSpanPath := vdScope["default_span_path"].(string)
+		defaultSpanPath := d.Get("default_span_path").(string)
+		if defaultSpanPath != "" {
```

Comment:
If the user provides an empty string for default_span_path we will not send a default span in the payload to NSX.
Then NSX will set automatically /infra/network-spans/default.

Will this cause a permadiff?